cetmix_tower_server/tests/test_variable_option.py

```diff
@@ -0,0 +1,76 @@
+from odoo.tests import TransactionCase
```

Comment:
You should import 'common' instead, because it has all the pre-defined assets you can use for tests.

Reply:
I will make a change

Reply:
This is not done. You can simply check how it's done in other tests in the same(!) module, eg [here](https://github.com/cetmix/cetmix-tower/blob/1b9121272c56eb300f3a65704ddcbf6572c76df3/cetmix_tower_server/tests/test_command_log.py#L3)